signer.go

```diff
@@ -102,3 +103,8 @@ func signHost(ca tls.Certificate, hosts []string) (cert *tls.Certificate, err er
 		PrivateKey:  certpriv,
 	}, nil
 }
+
+func init() {
+	// Avoid deterministic random numbers
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Is that the canonical Go way to seed rand?

Reply:
I'm not sure if there is a canonical way. We do not need cryptographically strong random numbers (otherwise we'd use `crypto/rand`), sowe just need to make it unlikely that two consecutive runs of the code will produce the same string of random numbers.

If you look in the second example on https://golang.org/pkg/math/rand/, you'll see the following comment:

```go
	// Typically a non-fixed seed should be used, such as time.Now().UnixNano().
```